acceptance-tests/src/test/kotlin/com/chutneytesting/acceptance/tests/actions/AssertionsTaskAcceptanceTest.kt

```diff
@@ -0,0 +1,127 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.acceptance.tests.actions
+
+import com.chutneytesting.acceptance.common.checkScenarioSuccess
+import com.chutneytesting.acceptance.common.createScenario
+import com.chutneytesting.acceptance.common.executeScenario
+import com.chutneytesting.kotlin.dsl.ChutneyScenarioBuilder
+import com.chutneytesting.kotlin.dsl.Scenario
+
+val `Execution by UI controller` = Scenario(title = "Execution by UI controller") {
+  micrometerScenario(
+    """
+        {
+            "when":{
+                "sentence":"Put values in context",
+                "implementation":{
+                    "task":"{\n type: context-put \n inputs: {\n entries: {\n test_int: {\"status\":{\"code\":200.0,\"reason\":\"OK\"}} \n test_xml: <test><xml attr=\"attributeValue\"><status><code>200.0</code><reason>OK</reason></status></xml></test> \n test_xml_ns: <tt:test xmlns:tt=\"http://test.org\" xmlns=\"http://test.org\"><xml attr=\"attributeValue\"><status xmlns=\"http://simple.org\"><code>200.0</code><reason>OK</reason></status></xml></test> \n} \n} \n}"
+                }
+            },
+            "thens":[
+                {
+                    "sentence":"Json assert",
+                    "implementation":{
+                        "task":"{\n type: json-assert \n inputs: {\n document: \${'$'}{#test_int} \n expected: {\n ${'$'}.status.code: 200.0 \n ${'$'}.status.reason: OK \n} \n} \n}"
```

Comment:
I would underline the spEL use by `${"test_int".spEL}` instead of `${'$'}{#test_int}`.
Or maybe write an extension function for spEL in hjson like `val String.hjsonSpEL()` to include the backslash ?
This pattern appears a lot of time elsewhere.

The two others `${'$'}` seems not mandatory, after testing locally. This could be the case elsewhere.